src/routes/wohnen/components/OfferListItem.js

```diff
@@ -1,21 +1,54 @@
 // @flow
 
 import React from 'react'
-import style from './OfferListItem.css'
-import WohnenOfferModel from '../../../modules/endpoint/models/WohnenOfferModel'
+import WohnenOfferModel from 'modules/endpoint/models/WohnenOfferModel'
+import WohnenFormData from 'modules/endpoint/models/WohnenFormData'
+import styled from 'styled-components'
+import ListElement from 'modules/common/components/ListElement'
 
 type PropsType = {
-  offer: WohnenOfferModel
+  offer: WohnenOfferModel<*>
```

Comment:
What is the meaning of the "*"?

Reply:
It allows says that the type of the generic is not yet specified. You can make a check on a property which has the generic in its signature to specify it.